spec/sidekiq/evss/disability_compensation_form/submit_form526_all_claim_spec.rb

```diff
@@ -56,8 +58,12 @@
 
     before do
       cassettes.each { |cassette| VCR.insert_cassette(cassette) }
-      Flipper.disable(ApiProviderFactory::FEATURE_TOGGLE_RATED_DISABILITIES_BACKGROUND)
-      Flipper.disable(ApiProviderFactory::FEATURE_TOGGLE_RATED_DISABILITIES_FOREGROUND)
+      allow(Flipper).to receive(:enabled?).with(ApiProviderFactory::FEATURE_TOGGLE_RATED_DISABILITIES_BACKGROUND,
+                                                instance_of(OpenStruct)).and_return(false)
```

Comment:
I recommend replacing `instance_of(OpenStruct)` with `anything` (which is the rspec matcher for... anything) since it's shorter, more readable, and less brittle in case the app code changes to use something other than an OpenStruct.